### ISSUE TITLE
Doesnt launch automatically captive portal in Android

### ISSUE THREAD

**Issue by SipriusPT** [wifiphisher/wifiphisher#1160]
Wifiphisher: `commit 53779384d09f3039b2ec897c7c91079c57965ae0`
Kernel: kali 4.18.0-kali2-amd64
Android: 8.0.0

It doesnt display automatically any popup when connect for the first time on my android, but if I try to open in browsers any website, it does the redirection for the phishing page, who is in this case is the 10.0.0.1:8080.

This is the command that I am using:

`wifiphisher -i wlan0 -nD --logging -cP /root/Documents/wifiphisher_logs/wifiphisher.log`

And then choose the target WIFI and phishing page type, where I have tested with all the available but same output, no popup.

Not sure if its a missing argument, or a possible issue.

**Comment by Bsoft2007** [comment]
Boas, no android deve aparecer nas notificacoes a rede fake, que ao clicares la vai directo para o portal cativo.

**Comment by failover88** [comment]
Hello, it doenst show in android notifications any new entrie, and in a system with a captive portal it is supposed to open at least a new window or popup, automatically, when it joins such network. In this case, with this setup, that window or popup doesnt show up.

**Comment by himuura** [comment]
Agreed, both android and win 8.1 with FF or Chrome to the latest dont open the captive portal with the Firmware Upgrade template...dunno what's happening...is there a --debug or --verbose mode?

**Comment by himuura** [comment]
nevermind, i was using wifiphisher -nD -iI wlan0 and that didn't even opened the website.
